From 1e5032696c93619536af6ce6b05d8182f099c6b9 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Fri, 2 Oct 2020 17:09:57 +0800
Subject: [PATCH] ARM: dts: rv1126-bat-ipc-v10: add some optional properties

Keep 0dB for adc/dac volume, 6(18dB) for adc-pga-gain,
and using hardware loopback which is dac_l+adc_r.

Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
Change-Id: I713e25984cfe20f3e50d14d381a6825e48d5296b
---
 arch/arm/boot/dts/rv1126-bat-ipc-v10.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
index d75977e6fe4d..624a026ef20c 100644
--- a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
+++ b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
@@ -235,6 +235,10 @@
 		reg = <0x18>;
 		clocks = <&cru MCLK_I2S0_TX_OUT2IO>;
 		clock-names = "mclk";
+		adc-pga-gain = <6>;  /* 18dB */
+		adc-volume = <0xbf>;  /* 0dB */
+		dac-volume = <0xbf>;  /* 0dB */
+		aec-mode = "dac left, adc right";
 		pinctrl-names = "default";
 		pinctrl-0 = <&i2s0m0_mclk &spk_ctl>;
 		assigned-clocks = <&cru MCLK_I2S0_TX_OUT2IO>;
-- 
2.35.3

